### ISSUE TITLE
Animated Gif icons: stop

### ISSUE THREAD

**Issue by shawnthompson** [wet-boew/wet-boew#8639]
Would be nice to have a button in the skip navigation or something to stop all animation on the site, Like animated gifs, for those who want to put in memes LOL

Something like [togglific.io](https://togglific.io/)

I also found this on Codeopen:

https://codepen.io/zjun/pen/dGPqzQ

**Comment by thekodester** [comment]
Something that might be an issue is if someone is using FontAwesome animations which work with CSS.

**Comment by juleskuehn** [comment]
Are there any examples of GoC websites (using WET) that have these issues? As a starting point for testing togglific etc.

Re: FontAwesome (which is consistently a problem for accessibility) the codepen example Shawn linked to might work. [This discussion on stack overflow](https://stackoverflow.com/questions/45343768/how-to-set-animated-icons-to-stop-after-some-time) suggests removing the fa-spin class.

Between those two approaches, it should be doable! One thing to keep in mind is that the animation might convey information that the stopped icon does not (ex. a spinning round arrow indicates loading, but the same image stopped suggests a "refresh" button). That is something that the web designer / content author using animations would have to consider.

**Comment by RobJohnston** [comment]
I'm no front-end guru, but I'm now skimming the [Bootstrap accessibility doc ](https://getbootstrap.com/docs/4.3/getting-started/accessibility/#reduced-motion)that says it supports the [prefers-reduced-motion](https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-motion) media feature. Useful?

**Comment by thekodester** [comment]
You'll need to confirm if that exist in Bootstrap 3, as the link you shared is for Bootstrap 4.

**Comment by thekodester** [comment]
I don't see it mentioned in Bootstrap 3, but something that could be custom coded. See https://getbootstrap.com/docs/3.4/getting-started/#accessibility.
